### ISSUE TITLE
No way to inspect checkbox value?

### ISSUE THREAD

**Issue by gpascale** [mlaursen/react-md#422]
### Description

I may be dense, but is it actually impossible to inspect the value of a checkbox? Some other components have ways to inspect the value, but they all seem to work a little differently

e.g.
.getField().value for a SelectField
.state.value for an Autocomplete
??? for a Checkbox

I know that I can add an onChange handler and track the state myself, as in the examples, but that's a real pain in this case. I have many checkboxes and a form-style input. That is to say I only need their values when the user presses a submit button. Having to declare my own state and write onChange handlers seems like a lot of extra work when all I really need is an "isChecked" attribute from the checkbox.

Am I missing something here? What is the recommended approach?

Thanks,
Greg

**Comment by mlaursen** [comment]
Oh. Right now there isn't. In 1.1.x there will be just `checkboxRef.value` `checkboxRef.checked` which will be accessible.

In this case, you could always do
```js
const checkboxes = document.querySelectorAll('input[name="checkboxes-for-submit"]');
const values = [...checkboxes].map(({ value }) => value);
// or
const selectedBoxes = document.querySelectorAll('input[name="checkboxes-for-submit"]:checked');
const selectedValues = [...selectedBoxes].map(({ value }) => value);
```

**Comment by gpascale** [comment]
Thanks for this. I'm glad to hear it will be addressed in 1.1!
